tests/e2e/solidity/suites.go

```diff
@@ -92,14 +92,25 @@ var _ = utils.DescribePrecompile(func() {
 		gomega.Expect(running).Should(gomega.BeFalse())
 	})
 
+	ginkgo.It("reward manager", ginkgo.Label("solidity-with-npx"), func() {
+		err := startSubnet("./tests/e2e/genesis/reward_manager.json")
+		gomega.Expect(err).Should(gomega.BeNil())
+		running := runner.IsRunnerUp(grpcEp)
+		gomega.Expect(running).Should(gomega.BeTrue())
+		runHardhatTests("./test/ExampleRewardManager.ts")
+		stopSubnet()
+		running = runner.IsRunnerUp(grpcEp)
+		gomega.Expect(running).Should(gomega.BeFalse())
+	})
+
 	// ADD YOUR PRECOMPILE HERE
 	/*
 			ginkgo.It("your precompile", ginkgo.Label("solidity-with-npx"), func() {
 			err := startSubnet("./tests/e2e/genesis/{your_precompile}.json")
 			gomega.Expect(err).Should(gomega.BeNil())
 			running := runner.IsRunnerUp(grpcEp)
 			gomega.Expect(running).Should(gomega.BeTrue())
-			runHardhatTests("./test/Example{YourPrecompile}Test.ts")
```

Comment:
Should this be reverted back since we still use `Example{...}` in most of our file names?

Reply:
This is not a hard requirement. There is also `ERC20NativeMinter`, so I did not want to imply it should start with `Example`.